_data/ph_authors.yml

```diff
@@ -1357,6 +1357,22 @@
     - editorial
     - proghist
   status: volunteer
+
+- name: Josir Cardoso Gomes
+  team: true
+  orcid: 0000-0001-7629-1404
+  bio:
+      en: |
+        Josir Gomes is a PhD student in Information Science at PPGCI/IBICT/UFRJ. Graduated in Computer Science. Information Systems Developer, acting mainly with relational databases, data mining and free software.
+      pt: |
+        Josir Gomes é Doutorando em Ciência da Informação no PPGCI/IBICT/UFRJ. Graduado em Processamento de Dados e especialista em Sistemas de Informação, Banco de Dados Relacionais, Mineração de Dados e Software Livre.
+      es: |
+        Josir Gomes es estudiante de Doctorado en Ciencias de la Información en PPGCI/IBICT/UFRJ. Se graduó en Procesamiento de Datos y es especialista en Sistemas de Información, Bases de Datos Relacionales, Minería de Datos y Software Libre.
+      fr: |
+        Josir Gomes est doctorant en sciences de l'information au PPGCI/IBICT/UFRJ. Diplômé en informatique et spécialisé dans les systèmes d'information, les bases de données relationnelles, le data mining et les logiciels libres.
```

Comment:
Josir Gomes est doctorant en sciences de l'information au PPGCI/IBICT/UFRJ. Diplômé en informatique, il se spécialise dans les systèmes d'information, les bases de données relationnelles, l'exploration de données et les logiciels libres.

Reply:
Thank you!